packages/kernel-5.10/config-bottlerocket-metal

```diff
@@ -37,6 +42,10 @@ CONFIG_NET_VENDOR_MELLANOX=y
 CONFIG_MLX5_CORE_EN=y
 CONFIG_NET_SWITCHDEV=y
 
+# Cisco UCS HBA support
+CONFIG_FCOE_FNIC=m
+CONFIG_SCSI_SNIC=y
```

Comment:
This also needs to be `y`? Why is one a module and not the other?

Reply:
To be completely hones I am not sure on this one. My reasoning here would have been that someone might want to use an sNIC device for booting. If that is at all possible is not completely clear to me glancing over Cisco documentation.

I am happy to "demote" this to a module instead of built-in and change that once it becomes a need.

Reply:
> I am happy to "demote" this to a module instead of built-in and change that once it becomes a need.

Let's do that - I am also not sure how it'd work to boot from a device like that, and if GRUB would even support it.